index.html

```diff
@@ -1003,6 +1001,545 @@ <h4><a href="#" class="link-dark nav-link p-0 mt-2"> Hokkaido: Japan’s Norther
+                        <div class="clearfix">
+                            <h5><a href="#" class="link-dark nav-link p-0 mt-2 fw-bold font-size">'Tiger King' Zoo Owners Ordered To Hand Over Big Cats to Government</a></h5>
+                        </div>
+                    </article>
+                </div>
+            </section>
+            <div class="gray w-100 d-md-none"> <br></div>
+            <div class="row ">
+             <!--Subscribe-->
+            <section class="my-3 mx-2 border-1 col-lg-7">
+                <header><h5 class="otext fw-bold">SUBSCRIBE</h5></header>
+                <div class="bg-light jumbotron mt-3 py-5">
+                    <div class="d-flex flex-column align-items-center flex-md-row align-items-md-end">
+                        <div class="position-relative p-3">
+                            <img class="" src="https://web.archive.org/web/20210115135144im_/https://d.newsweek.com/en/full/1700315/january-15-2021-cover.jpg?w=170&q=95&f=02cc1a3558f7d7b001baea3b926907e7" alt="image" width="170" height="230">
+                            <img class="position-absolute subimage2" src="https://web.archive.org/web/20210122003009im_/https://d.newsweek.com/en/full/1704591/cover-january-22-2021.jpg?w=170&q=95&f=ee95940a99c02547a13c42e35dbfbc84" alt="image" width="170" height="230">
+                            <div class="subdiv2"></div>
+                        </div>
+                        <div class="text-box d-flex flex-column align-items-center align-items-md-start p-4">
+                            <strong class="fs-4">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</strong>
+                            <p>All Access + Weekly Delivery.
+                                Choose A Membership That's Perfect for You!</p>
+                            <button class="btn first-header text-white fw-bold">SUBSCRIBE &gt;</button>
+                        </div>
+                    </div>  
+                </div>                        
+            </section>
+            <div class="gray w-100 d-md-none"> <br></div>
+            <!--Newsletter-->
+            <section class="mt-3 mx-2 border-1 col-lg-4 ">
+                <header><h5 class="otext fw-bold">NEWSLETTER</h5></header>
+                <div class="d-md-flex flex-md-row mt-lg-3 letter">
+                    <div class="phone"></div>
+                    <div class="border border-1 mb-4 ">
+                     <form class=" my-3 mx-2 px-lg-5">
+                            <div class="text-center text-lg-start">
+                              <h5 class="fw-bolder text-uppercase mt-lg-5">The Starting 5</h5>
+                              <p class="sm-font1 mt-lg-3">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                              <div class="emaildiv">
+                              <input type="text"  class="email2 form-text border-light border-1" placeholder="Email address">
```

Comment:
```suggestion
                              <input type="email"  class="email2 form-text border-light border-1" placeholder="Email address">
```
- [ ] You should use the input type `email ` for the email input fields.

Reply:
resolved